src/style.css

```
@@ -1 +1,61 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+}
+
+.img {
+  display: block;
+}
+
+.header {
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+}
+
+.logo {
+  padding: 10px 0 10px 50px;
+}
+
+.nav {
+  padding-right: 50px;
```

```suggestion
  padding: 10px 0 10px 50px;
}

.nav {
  padding-right: 50px;
```

Щоб не задавати в двох місцях бокові падінги по 50px, можна самому хедеру задати 
```
padding: 0 50px;
```